docs/changelog.rst

```diff
@@ -2,6 +2,7 @@
 Changelog
 =========
 
+* :bug:`-` Tokens with large amount or decimal value are calculated with full precision.
```

Comment:
probably is better to not add this in the docstring. The working has to change because as a user I don't know what it means

Reply:
yes this should not have a changelog